subprojects/dependency-management/src/test/groovy/org/gradle/internal/component/model/AttributeConfigurationSelectorTest.groovy

```diff
@@ -489,7 +489,7 @@ All of them match the consumer attributes:
         Stub(ModuleConfigurationMetadata) {
             getName() >> name
             getAttributes() >> attributes
-            getCapabilities() >> Mock(CapabilitiesMetadata) {
+            getCapabilities() >> Mock(CapabilitiesMetadataInternal) {
```

Comment:
instead of using a mock here, could this just be a real implementation?

Reply:
That works.